README.md

```diff
@@ -97,147 +97,136 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
+### 0.75 - October 2023 Update
 
 In this release, we focused on stability and improvements.
 
 **Highlights**
 
- - Upgraded to Windows App SDK 1.4.1, increasing stability of WinUI3 utilities. Thanks [@dongle-the-gadget](https://github.com/dongle-the-gadget) for starting the upgrade!
- - Text Extractor was upgraded to its version 2.0, with a new overlay, table mode and more Quality of Life improvements. Thanks [@TheJoeFin](https://github.com/TheJoeFin)!
- - Improved FancyZones stability, fixing some layout resets and improving handling of newly created windows on Windows 11.
- - Fixed many silent crashes that were reported to Watson and the user's event viewer.
+ - New utility: An environment variables editor that also allows configuring profiles that can be enabled/disabled. Thanks [@niels9001](https://github.com/niels9001) for the design and UI work that made this possible!
+ - Settings has a new Dashboard home page, with quick access for enabling modules, short descriptions and activation methods. Thanks [@niels9001](https://github.com/niels9001) for the design and UI work that made this possible!
+ - Added a previewer to Peek that hosts File Explorer previewers to support every file type that a machine is currently able to preview. For example, this means that if Microsoft Office handlers are installed, Peek can preview Office files. Thanks [@dillydylann](https://github.com/dillydylann)!
 
 ### General
 
- - Turning animations off in Windows Settings will now also turn them off in PowerToys.
- - Upgraded the Windows App SDK dependency to 1.4.1. Thanks [@dongle-the-gadget](https://github.com/dongle-the-gadget) for the original 1.4.0 upgrade!
- - Show in the thumbnail label and application titles when running as administrator. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
- - Upgraded the Win UI Community Toolkit dependency to 8.0. Thanks [@niels9001](https://github.com/niels9001)!
-
-### Awake
-
- - Added down-sampled variants to the application's icon. Thanks [@morriscurtis](https://github.com/morriscurtis)!
+ - Many typo fixes through the projects and documentation. Thanks [@brianteeman](https://github.com/brianteeman)!
+ - Refactored and improved the logic across utilities for bringing a window to the foreground after activation.
 
 ### Color Picker
 
- - After adding a new color in the editor, the history will scroll the new color into view. Thanks [@peerpalo](https://github.com/peerpalo)!
+ - Terminate a picker session by using the right mouse button. Thanks [@fredso90](https://github.com/fredso90)!
 
-### Crop and Lock
- - Fixed a Crop and Lock crash that would occur when trying to reparent a window crashes the target application. An error message is shown instead.
+### Environment Variables
+ - Added a new utility: An environment variables editor that also allows configuring profiles that can be enabled/disabled. Thanks [@niels9001](https://github.com/niels9001) for the design and UI work that made this possible!
```

Comment:
```suggestion
 - Added a new utility: An environment variables editor that has the functionality to profiles that can be enabled/disabled. Thanks [@niels9001](https://github.com/niels9001) for the design and UI work that made this possible!
```

Same as above

Reply:
Ended up doing it with a commit, since the suggestion was missing the "configure" word. Thank you ;)